tests/puppeteer/campaigns/sanity/01_installShop/01_installShop.js

```diff
@@ -27,39 +29,45 @@ describe('Install Prestashop', async () => {
   // Steps
   it('should open the Install page', async function () {
     await this.pageObjects.installPage.goTo(global.INSTALL.URL);
-    await this.pageObjects.installPage.checkStepTitle(this.pageObjects.installPage.firstStepPageTitle,
+    const result = await this.pageObjects.installPage.checkStepTitle(this.pageObjects.installPage.firstStepPageTitle,
       [this.pageObjects.installPage.firstStepFrTitle, this.pageObjects.installPage.firstStepEnTitle]);
```

Comment:
I'm having a hard reading this, maybe it would be nice to pass parameters on several line?

A suggestion (not sure how to deal with the pageTitle array):

```suggestion
const result = await this.pageObjects.installPage.checkStepTitle(
        this.pageObjects.installPage.firstStepPageTitle,
        [
            this.pageObjects.installPage.firstStepFrTitle,
            this.pageObjects.installPage.firstStepEnTitle
        ]
 );
```